application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -147,34 +134,29 @@ static ReminderRoute toPrivate(PrivateChannel channel) {
         }
     }
 
-    private void reminderFailureAction(int id, long channelId, long authorId,
-                                       CharSequence content, long guildId, int reminderFailureCounter) {
-        if (reminderFailureCounter <= 3) {
-            Instant current = Instant.now();
-            database.write(context -> context.newRecord(PENDING_REMINDERS)
-                    .setId(id)
-                    .setCreatedAt(Instant.now())
-                    .setGuildId(guildId)
-                    .setChannelId(channelId)
-                    .setAuthorId(authorId)
-                    .setRemindAt(current.plusSeconds(60))
-                    .setContent(content.toString())
-                    .setReminderFailureCounter(reminderFailureCounter + 1)
-                    .insert());
-        } else {
+    private void attemptRetryReminder(Reminder reminder) {
+        if (!(reminder.failureAttempts() <= MAX_FAILURE_RETRY)) {
             logger.warn(
                     """
-                            Failed to send a reminder with (authorID '{}'), skipping it. This can be due to a network issue, \
+                            Failed to send a reminder with (authorID '{}') and (content '{}'), skipping it. This can be due to a network issue, \
```

Comment:
personally, id put the content at the very end of the log message. the content can be really long text and its hard to read the log message if its put right into the sentence

Reply:
also, it would be helpful to have the reminder ID, not just the author ID

Reply:
yea kinda makes sense, didn't think like that because never really used logging.

Reply:
fixed this